### ISSUE TITLE
TDML generator

### ISSUE THREAD

**Issue by jw3** [jw3/example-daffodil-vscode#44]
Function that will roll together a schema and data sample into a TDML file. 

This will be useful when submitting questions or bugs to the community.

**Comment by arosien** [comment]
Additionally, what about taking a TDML file as an input, showing the available schemas and data to choose from in order to debug?

**Comment by jw3** [comment]
Yeah that's good.  I updated the title to represent a more general TDML capability here.  More fine grained issues can be added later as needed.
